.webpack/webpack.common.js

```diff
@@ -71,8 +71,7 @@ const config = {
       bourbon: 'bourbon.scss',
       'plotly-basic': 'plotly.js-basic-dist',
       'plotly-gl2d': 'plotly.js-gl2d-dist',
-      'd3-scale': path.join(projectRootDir, 'node_modules/d3-scale/dist/d3-scale.min.js'),
-      printj: path.join(projectRootDir, 'node_modules/printj/dist/printj.min.js'),
+      printj: 'printj/printj.mjs',
```

Comment:
Would it make sense to merge #7038 first?

Reply:
Yes, I'll definitely be taking another look at your PR this week as I'm diving into our build/packaging process. I may split the webpack changes out of this PR, which is kind of experimental at this point anyway. Thanks for your patience again, really appreciate it.

Reply:
Np 🤗